DataManagementSystem/Client/FileCatalogClientCLI.py

```diff
@@ -412,25 +428,60 @@ def removeFile(self,args):
     except Exception as x:
       print("Error: rm failed with exception: ", x)
       
-  def removeDirectory(self,args):
-    """ Remove file from the catalog
-    """  
-    
-    path = args[0]
-    lfn = self.getPath(path)
-    print("lfn:", lfn)
+  def removeDirectory(self,lfn, recursive=False, forceNonEmpty=False):
+    """ Remove a given directory from the catalog. Remove multiple directories
+        recursively if recursive flag is True. Remove contained files if forceNonEmpty
+        flag is true.
+    """
+
+    if recursive or forceNonEmpty:
+      resultListDirectory = self.fc.listDirectory(lfn, False)
+      if not resultListDirectory['OK']:
+        print('Failed to look up the directory contents')
```

Comment:
Why `print` and not gLogger?

Reply:
This was like that since ever in all the CLIs, I think this is OK,